run.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/bash
 
-babel src --out-dir build --copy-files
+node_modules/babel-cli/bin/babel.js src --out-dir build --copy-files
```

Comment:
I think there's a better way to handle this, probably by adding an npm lifecycle script (e.g. "compile") which will look inside that directory for the babel executable, but that's okay for now.